tests/test_visitors/test_ast/test_classes/test_methods/test_useless_overwriting_method.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+
+from typing import List
+
+import pytest
+
+from tests.test_visitors.test_ast.test_classes.test_methods.data_model import (
+    TestMethodArgs,
+)
+from wemake_python_styleguide.violations import oop
+from wemake_python_styleguide.visitors.ast.classes import WrongMethodVisitor
+
+_regular_method_detailed = """
```

Comment:
Our naming convention applies that it should be `regular_method_detailed` and `regular_method_short`